cyclonedds_config.xml

```diff
@@ -0,0 +1,18 @@
+<!-- <?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
its a different version, the uncommented one works for now but I get a deprecation warning on a specific version of ros (can't remember if it's foxy or humble). The commented out version uses what I found in cyclone documentation but it doesn't work for the ros version that gets deprecation messages. I left the one that works, and the commented one in case we need it later on